Formula/jfrog-cli-go.rb

```diff
@@ -31,8 +31,8 @@ def install
   end
 
   test do
-    actual = pipe_output("#{bin}/jfrog rt -v")
-    expected = "jfrog version 1.0.1\n"
+    actual = pipe_output("#{bin}/jfrog -v")
+    expected = "jfrog version 1.1.0\n"
     assert_equal expected, actual
```

Comment:
Change this to: `assert_equal "jfrog version #{version}", shell_output("#{bin}/jfrog rt -v").chomp`. If that fails `brew test` locally you may need to add a second argument to `shell_output` to specify the exit code.